easybuild/easyblocks/i/impi.py

```diff
@@ -50,6 +50,8 @@ class EB_impi(IntelBase):
     @staticmethod
     def extra_options():
         extra_vars = {
+            'rebuild_libfabric': [True, 'Rebuild the internal shipped libfabric instead of using the provided binary', CUSTOM],
```

Comment:
@damianam please fix long line, for example as follows:

```python
def extra_options():
        extra_vars = {
            'rebuild_libfabric': [True, "Rebuild the internal shipped libfabric "
                                        "instead of using the provided binary", CUSTOM],
            ...
```

Also, please move `rebuild_libfabric` down (keep these sorted alphabetically).

Bonus points if you can come up with a naming that keeps both new parameters together (`libfabric_rebuild` maybe?)...